### ISSUE TITLE
Option for condensed notifications

### ISSUE THREAD

**Issue by holljen** [integrations/microsoft-teams#34]
We have a (rather long) template for issues and PR'r. This makes notifications take up a lot of space, half a screen on my laptop. In practice this makes the channel unreadable.

It would be nice if it was possible to turn on some option to only display the "subject" of the notifications. 

If it was possible to have some sort of hide/show option for the rest of the PR, ticket etc. that would be even better!

**Comment by ashokirla** [comment]
For long notification cards, Teams platform already has inbuilt wrapping experience. So if any card is beyond 6-7 lines, it should get wrapped. Is that not helping in your case.

**Comment by holljen** [comment]
How is that supposed to look? Most of my cards are more than 6-7 lines and there is no wrapping (collapsing) of that. Do I have to turn that on somewhere?

**Comment by holljen** [comment]
![Screenshot 2020-09-11 at 21 44 12](https://user-images.githubusercontent.com/11437018/92966581-17c10500-f478-11ea-9ae8-bf455c202f1e.png)
Just to show you what my channel looks like....

**Comment by ashokirla** [comment]
Okay, we will check with the platform team. Ideally, it should wrap and say see more.

**Comment by ashokirla** [comment]
@DivyankaM Can you look into this issue.

**Comment by gauravsaralMs** [comment]
we have an issue opened with team platform for this:
https://github.com/microsoft/AdaptiveCards/issues/4583

**Comment by holljen** [comment]
Any progress on this? It makes the whole integration useless for us. We have disabled it until this gets sorted.

**Comment by tannerbaum** [comment]
Agreed that this makes the integration somewhat unusable for us as well. 

<img width="1346" alt="Screen Shot 2022-03-02 at 14 47 37" src="https://user-images.githubusercontent.com/1280597/156373796-bade1a43-02fa-44a3-89f7-cec21a5a9114.png">
